src/main/java/com/epam/HelloController.java

```diff
@@ -10,6 +11,7 @@
 
   @RequestMapping(method = RequestMethod.GET)
   public String hello() {
+    LoggerFactory.getLogger(HelloController.class).warn("hell");
```

Comment:
I guess it was just for test and it will be removed after some time. So I suggest you move it to simple unit-test (just for example of using logger for the team) and the second point is removing import `import org.slf4j.LoggerFactory;` and just add annotation `@Slf4j` to class and please use var `log ` in code - `log.warn("hell");`